apis/mutations/v1alpha1/value_conversion.go

```diff
@@ -0,0 +1,29 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"github.com/open-policy-agent/gatekeeper/apis/mutations/unversioned"
+	conversion "k8s.io/apimachinery/pkg/conversion"
+)
+
+func Convert_unversioned_AssignField_To_v1alpha1_AssignField(in *unversioned.AssignField, out *AssignField, s conversion.Scope) error { // nolint:revive
```

Comment:
Are we just using the auto-generated conversion functions here? If so, does this file need to exist?

Reply:
This is a function that I have to add manually because `apis/mutations/v1alpha1/zz_generated.conversion.go` didn't generate a conversion function for `AssignField` as the v1alpha1 `AssignField` type is missing the `ExternalData` field. This conversion function will simply drop the `ExternalData` field in `unversioned` and copy the rest of the data to a v1alpha1 `AssignField` pointer.

Reply:
Per above comment referencing k8s backwards compatibility guidelines, both v1alpha1 and v1beta1 should support external data.